GirafIntegrationTest/tests/test_user_controller.py

```diff
@@ -401,6 +401,7 @@ def test_user_can_check_multiple_settings(self):
         self.assertEqual(3, response['data']['theme'])
         self.assertEqual(2, response['data']['nrOfDaysToDisplay'])
         self.assertTrue(True, response['data']['greyScale'])
+        self.assertTrue(True, response['data']['LockTimerControl'])
```

Comment:
This always passes since the asserted expression is `True`. 
Look at the method definition: `assertTrue(expr, msg=None)`
```suggestion
        self.assertTrue(response['data']['greyScale'])
        self.assertTrue(response['data']['LockTimerControl'])
```